libs/ui-watt/src/lib/components/tooltip/watt-tooltip.component.scss

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use "sass:math";
+@use "@energinet-datahub/watt/utils" as watt;
+
+$arrow-size: 8px;
+$offset: 6px;
+
+:host {
+  z-index: 500;
+  display: inline-block;
+  background: var(--watt-color-primary-dark);
+  color: var(--watt-color-primary-dark-contrast);
+  line-height: 18px;
+  border-radius: 6px;
+  @include watt.space-inset-squish-s;
+  @include watt.typography-watt-text-s;
+  opacity: 0;
+  transition: opacity 0.3s ease-in-out;
```

Comment:
issue: some of the arrows (e.g. `top-start` or `top-end`) are misaligned.

![image](https://user-images.githubusercontent.com/1096332/200314677-4bb70a3f-5d4e-4818-933a-26ac58b28eb1.png)

Reply:
Should be fixed now :)